test/netlib/mat_convert.py

```diff
@@ -0,0 +1,68 @@
+import scipy.io
```

Comment:
comments of the library in the begining of files needed

Reply:
Thanks! The script has been now refactored.